### ISSUE TITLE
Create or download an index for "stratospheric warming". 

### ISSUE THREAD

**Issue by ribeiron** [oceanhackweek/ohw23_proj_SAupwelling#6]
```[tasklist]
### Tasks
```

**Comment by ribeiron** [comment]
What I do know: 
THE SUPPRESSION OCCURS WHEN THERE IS A WEAKENING OF THE SOUTHEASTERLY WINDS. 

Possible causes for that: 
- Southern Stratospheric Warming 
- Fronts coming from the south might also end suppress the upwelling. 
- Weakening of the polar vortex causing cabelling. 

Other options: 
- Marine Heatwave from Northwest AUS in 2013 (Maggiorano et al., 2021). Could that have been advected by the Leuwin current?
- A really hot summer?
- PDO?

**Comment by ribeiron** [comment]
I have looked into Stratospheric Warming and found the information that it’s only happened twice since 1980. 2002 and 2019
The true definition is a wind reversal at 10hpa.
Here is an article from ABC from the last time it happened: https://www.abc.net.au/news/2019-09-06/rare-weather-event-over-antarctica-drives-hot-outlook/11481498
Note that "SSW is rare in the southern hemisphere with only one major event ever identified, in 2002 — one of Australia's driest years on record. Dr Hendon said similar, less intense stratospheric warmings had been linked to other dry years in Australia."

What I am reading is that, although we know that sudden stratospheric warming events cause weak vortex in spring, we can get a weaker vortex without a full SSW. So I went on to pursue a polar vortex index. I got 3 different ones: 
- Stratospheric-Tropospheric Couple Mode Index (Lim et al., 2018)
- JRA55-derived
- Breakdown Index (Black & Daniel)

The notebook can be found here: https://github.com/oceanhackweek/ohw23_proj_SAupwelling/blob/main/Notebooks/AntarcticPV_timeseries.ipynb

The next step is to compare these indexes to the actual mooring data from SA to see if we find anything interesting there. 

This issue is done, so I will be closing it now.
